opentype/loader/writer.go

```diff
@@ -0,0 +1,85 @@
+package loader
+
+import (
+	"encoding/binary"
+	"math"
+)
+
+// Table is one opentype binary table and its tag.
+type Table struct {
+	Content []byte
+	Tag     Tag
+}
+
+// Write creates a single font file from the given [tables] slice,
+// which must be sorted by Tag
+func Write(tables []Table) []byte {
+	buffer := writeTTF(tables)
```

Comment:
This function has a single purpose - to call `writeTTF`, which is strange - if it is generic we don't need the extra function, but if it is because this writer is for TTF perhaps the generic name isn't right?

Reply:
Fair point. In the future, we may want to support other formats, like web font (WOFF). In such case, the Write function would become `Write(tables []Table, woff bool) []byte`, and internally call `writeTTF` or `writeWOFF`. 
But, for now, it seems indeed clearer to drop the internal function and rename `Write` to, say, `WriteTTF`.